### PR TITLE
Allow building on Windows (direct)

### PR DESCRIPTION
**Pull request by stephendade**
This PR allows users to build the Wiki directly on Windows - no WSL or Cygwin required.

### CODE REVIEW

**Comment by stephendade** [comment]
So, turns out this doesn't work on Python 3.10 on Windows. 3.8.x works though. Will investigate...

**Comment by Hwurzburg** [comment]
tried with Python 3.8.10    still get the same error on ../wiki directory and build stops before it really starts

**Comment by stephendade** [comment]
Ok, figured this out.  Python-on-Windows does not add the pip user folder to the system path. So I've modified the PR to install the pip packages to the Python install folder instead. Does require to user to do a "right click -> Run as admin" on the Sphinxsetup.bat.

**Review by Hwurzburg — APPROVED** [review]
setup a virgin VM installed WIN10....installed python 3.8.9 with path setup checked and allowed long paths when asked, installed git bash, cloned my repo in bash, pulled your pr, ran Sphinxsetup.bat from within bash....then built the DEV wiki....no issues

File: common/source/docs/common-wiki-editing-setup.rst [hunk 1]
```diff
@@ -52,6 +52,21 @@ Run the following command from the ardupilot_wiki directory you cloned:
 
 Then jump down to "Build the wiki".
 
+Setup in Windows
+----------------
+
+Running in WSL or Cygwin is not required. The ArduPilot wiki can be built directly on Windows.
+
+Ensure that the latest version of `Paython <https://www.python.org/downloads/>`__ is installed.
```
[the remark this answers is not in the document]
Comment (stephendade) [reply common/source/docs/common-wiki-editing-setup.rst]: Fixed

**Comment by kd0aij** [comment]
tested on native Win11 with Python 3.9.7
fresh clone of ardupilot_wiki
"right click -> Run as admin" on the Sphinxsetup.bat
"python --update.py --site plane"  appeared to run properly, with results in ../wiki/plane

@stephendade Should this also work with Python 3.10?

**Comment by stephendade** [comment]
> @stephendade Should this also work with Python 3.10?

Yes, it should

**Comment by kd0aij** [comment]
confirmed: put C:\Users\markw\AppData\Local\Programs\Python\Python310\python.exe at front of system and user paths
then reran Sphinxsetup.bat 

```
python .\update.py --site plane

...

build succeeded, 4 warnings.

The HTML pages are in build\html.

Build finished. The HTML pages are in build/html.
copy: plane
DEBUG: Creating backup
DEBUG: recreating C:\Users\markw\git\kd0aij\ardupilot_wiki\..\wiki\old
DEBUG: Moved to C:\Users\markw\git\kd0aij\ardupilot_wiki\..\wiki\plane
DEBUG: removing C:\Users\markw\git\kd0aij\ardupilot_wiki\..\wiki\old
```

results in ..\wiki\plane look OK

File: common/source/docs/common-wiki-editing-setup.rst [hunk 2]
```diff
@@ -41,8 +41,8 @@ Be sure to keep your fork of the repository updated, both locally, and on GitHub
        git rebase upstream/master
        git push -f origin master
        
-Setup in Linux or Windows WSL
------------------------------
+Setup in Linux
+-------------------------
```
Comment (Hwurzburg) [inline common/source/docs/common-wiki-editing-setup.rst:45]: one little nit and will merge....underlines must be same length as text above...thanks

**Review by Hwurzburg — CHANGES_REQUESTED** [review]

File: common/source/docs/common-wiki-editing-setup.rst (hunk 2 above)
Comment (stephendade) [reply common/source/docs/common-wiki-editing-setup.rst]: Fixed!

**Comment by Hwurzburg** [comment]
thanks...merging

**Pull request merged**